smartcontractkit/external-adapters-js
# BCI-995: Layer 2 Health Check -- update starknetjs

## Closes BCI-995

https://smartcontract-it.atlassian.net/browse/BCI-995

## Description

Update LAYER 2 Sequencer Health External Adapter to use starknet.js 5.x

## Changes

- Just updating starknet.js 5.x for an external adapter to the latest version



## Steps to Test

Run yarn jest unit

Also, you can run yarn @chainlink/layer2-sequencer-health-adapter start and see that the EA is healthy against the live Starknet sequencer

## Quality Assurance

N/A
